doc/cascadia/profiles.schema.json

```diff
@@ -602,6 +602,58 @@
         }
       ]
     },
+    "PrevTabAction": {
+      "description": "Arguments corresponding to a Previous Tab Action",
+      "allOf": [
+        { "$ref": "#/definitions/ShortcutAction" },
+        {
+          "properties": {
+            "action": { "type":"string", "pattern": "prevTab" },
+            "tabSwitcherMode": {
+              "oneOf": [
+                { "type": "null" },
+                {
+                  "enum": [
+                    "mru",
+                    "inOrder",
+                    "disabled"
+                  ],
+                  "type": "string"
+                }
+              ],
+              "default": null,
+              "description": "Got to the previous tab using \"tabSwitcherMode\". If no mode is provided, use the one globally defined one."
```

Comment:
```suggestion
              "description": "Move to the previous tab using \"tabSwitcherMode\". If no mode is provided, this action will use the one globally defined one."
```

Reply:
I guess it was supposed to be "Go". 🤦  "Move" is probably better.